gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/GfxdDDLMessage.java

```diff
@@ -151,99 +151,12 @@ protected void processMessage(final DistributionManager dm)
-                  processedIds.add(ddlId);
-                }
-              }
-            } finally {
-              memStore.releaseDDLReplayLock(true);
-            }
-          }
-          else {
-            // for ALTER TABLE we have to always wait for DDL replay to complete
-            // since we cannot pre-conflate the queue if CREATE is not done like
-            // for DROP
-            skipDDL = false;
-          }
-          if (!skipDDL) {
-            // ok, we do need to wait for DDL replay to complete;
-            // if DDL replay has reached the last iteration and is waiting
-            // for DD read lock, then skip wait on DD read lock by the node
-            // replay thread, if any, otherwise it can lead to a deadlock
-            // (#47873); this DDL message execution means that DD write lock is
-            // already held by the source node and the purpose of not allowing
-            // any more DDLs to come in by the DD read lock in replay thread is
-            // already served
-            memStore.setInitialDDLReplayWaiting(true);
-            try {
-              GemFireXDUtils.waitForNodeInitialization();
-            } finally {
-              memStore.setInitialDDLReplayWaiting(false);
-            }
-            // its possible that this DDL has been executed in replay
-            // by this time, so check again
-            final TLongHashSet processedIds = memStore.getProcessedDDLIDs();
-            synchronized (processedIds) {
-              skipDDL = processedIds.contains(ddlId);
-            }
-          }
-        }
-      }
-      if (skipDDL) {
-        SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_DDLREPLAY, toString()
-            + " Skipping execution of " + ddl.getValueToConflate()
-            + " due to replay in progress");
-        // Initial replay still in progress so just skip processing this message
-        // which will be later handled by replay; this breaks the behaviour of
-        // GfxdDDLMessage somewhat since a successful reply is sent without the
-        // DDL still having being executed but should have no improper behaviour
-        // as such; if we block, then DDLs on all processes will block till the
-        // initial replay of this VM is complete which is not desirable.
-
-        // store in pending message list in any case since put in DDL region
-        // will need to be done if origin node goes down before finish message
-        synchronized (pendingDDLMessages) {
-          // indicates that this node should not execute anything in finish
-          // message or if origin node goes down before finish message
-          this.args.connId = EmbedConnection.UNINITIALIZED;
-          pendingDDLMessages.put(ddlId, this);
-        }
-        throw new GfxdDDLReplayInProgressException(
-            "skipping DDL execution due to replay in progress");
-      }
+    // With the new logic of not allowing new ddls to be fired when replay is
```

Comment:
I will suggest commenting out the old code instead of removing. We don't know if customers may look for previous behaviour.